extensions/ql-vscode/src/databases/database-fetcher.ts

```diff
@@ -366,7 +370,11 @@ async function databaseArchiveFetcher(
   const unzipPath = await getStorageFolder(storagePath, databaseUrl);
 
   if (isFile(databaseUrl)) {
-    await readAndUnzip(databaseUrl, unzipPath, cli, progress);
+    if (origin.type === "testproj") {
```

Comment:
Would it make sense to change this condition to checking whether the `databaseUrl` is a directory on the filesystem? That seems like it should be more generic.

Reply:
OK. I'll change it to make it more consistent with `readAndUnzip` and `fetchAndUnzip`.